### PR TITLE
chore(e2e): fix report merging

### PR DESCRIPTION
**Pull request by binoy14**
### Description

This PR fixes the playwright report to include all versions of the shards based on browsers.

#### The raw report
Screenshot 2023-10-23 at 2 18 07 PM

#### HTML Report
Screenshot 2023-10-23 at 2 18 17 PM

### Notes for release

- N/A

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -65,6 +65,7 @@ jobs:
       - name: Run end-to-end tests
         env:
           SANITY_E2E_SESSION_TOKEN: ${{ secrets.SANITY_E2E_SESSION_TOKEN }}
+          PWTEST_BLOB_REPORT_NAME: ${{ matrix.project }}
```
Comment (bjoerge) [inline .github/workflows/e2e.yml:68]: is there any official docs about `PWTEST_BLOB_REPORT_NAME` we could link to here?

**Review by bjoerge — APPROVED** [review]
Nice - would be great to include a link to documentation for PWTEST_BLOB_REPORT_NAME, if possible.

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (binoy14) [reply .github/workflows/e2e.yml:68]: Unfortunately there is no docs for it, there is an open issue to update docs for adding the env var to the docs but I will link to the code in the meantime.

Comment (bjoerge) [reply .github/workflows/e2e.yml:68]: no worries 👍🏼

**Pull request merged**